src/lib/authenticator.ts

```diff
@@ -444,7 +444,7 @@ export class GenesysCloudClientAuthenticator {
 
     // Check if there is a token to test
     if (!this.authentications['PureCloud OAuth'].accessToken) {
-      throw new Error('Token is not set');
+      throw new Error('errorTokenNotSet');
```

Comment:
Changed error messages to be keys located within the translation files.

Reply:
My biggest reservation with this approach would be logging. Do we log these errors (to the server)? Any logs that go to sumo I'd prefer to be in english ;) A possible work-around to do what you're doing would be to create a `TranslatableError` which extends error and has a `translationKey` property. What are your thoughts about doing something like that to safeguard us for the future?

Reply:
Right now we aren't logging these to sumo. If we were it wouldn't go to sumo translated, just be translated in the React app. Instead it'd go up to sumo as `errorTokenNotSet` or whatever key is defined. I agree though, that would probably be a better approach.